### ISSUE TITLE
mitron [sf#78]

### ISSUE THREAD

**Issue by srynobio** [The-Sequence-Ontology/SO-Ontologies#78]
**Reported by cmungall on 2008-01-29 00:38 UTC**
the def of mitron seems at odds with its is\_a placement

is\_a SO:0000276 miRNA \[DEF: "Small, ~22-nt, RNA molecule that is the endogenous transcript of a miRNA gene. Micro RNAs are produced from precursor molecules \(SO:0000647\) that can form local hairpin structures, which ordinarily are processed \(via the Dicer pathway\) such that a single miRNA molecule accumulates from one arm of a hairpin precursor molecule. Micro RNAs may trigger the cleavage of their target molecules or act as translational repressors."\]
is\_a SO:0001034 mitron \[DEF: "An intron from whose debranched product an miRNA is derived."\]

**Comment by srynobio** [comment]
**Commented by eilbeck on 2008-01-30 20:59 UTC**
Logged In: YES 
user\_id=742851
Originator: NO

OK, so first of all there is a typo. It is mirtron.
This is really complicated. A debranched intron gets into the miRNA processing pathway.

I have made the term a child of intron\_domain.
I updated the spelling and the definiton.
"A debranched intron which mimics the structure of pre-miRNA and enters the miRNA processing pathway without Drosha mediated cleavage."

Now it does not have a relationship with miRNA.

\--K
